app/controllers/home_controller.rb

```diff
@@ -14,6 +14,33 @@ def index
     end
   end
 
+  def robots
```

Comment:
Dynamic is great but could increase the load. Should be monitored in production...

Reply:
We checked the server and found that we receive approximately 100 requests daily for the robots.txt file. This will not have a significant impact on performance. As you mentioned, we will monitor these requests and take proactive measures if intervention becomes necessary.